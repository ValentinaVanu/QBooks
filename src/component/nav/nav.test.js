import { QNav } from '.'
import renderer from 'react-test-renderer'


test('renders main nav component', () => {
  const nav = renderer.create(<QNav />).toJSON()
  expect(nav).toMatchSnapshot()
})
