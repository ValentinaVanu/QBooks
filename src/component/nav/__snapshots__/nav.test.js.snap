// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main nav component 1`] = `
<div
  className="sc-gsTCUz hMBXnp"
>
  <a
    className="sc-bdfBwQ hUbxGD"
    href="/item/1"
    onClick={[Function]}
  >
    item1
  </a>
  <a
    className="sc-bdfBwQ hUbxGD"
    href="/item/2"
    onClick={[Function]}
  >
    item2
  </a>
  <a
    className="sc-bdfBwQ hUbxGD"
    href="/item/3"
    onClick={[Function]}
  >
    item3
  </a>
</div>
`;
